server/server.js

```diff
@@ -0,0 +1,16 @@
+const express = require('express');
+const getAllCareers = require('../database/queries/get_all_careers');
+
+const app = express();
+
+app.get('/api/careers', (request, response) => {
+  getAllCareers((error, result) => {
+    if (error) {
+      console.log(error);
+    } else {
+      response.send(result);
```

Comment:
Same applies here. At the moment the response will be a pretty messy object that the front-end is left to deal with.

Reply:
I think we might play with this when the database is working